src/lib/dataEntities/networks.ts

```diff
@@ -135,7 +131,6 @@ export const l1Networks: L1Networks = {
     explorerUrl: 'https://etherscan.io',
     partnerChainIDs: [42161],
     blockTime: 14,
-    rpcURL: process.env['MAINNET_RPC'] as string,
```

Comment:
I've got a feeling these are used in the token-bridge-ui, we should take a look there

Reply:
yea but to instantiate a network you give a provider (in getNetwork)
so they already have the rpc lying around

Reply:
I don't think its useful for us to have the rpc url here since its never used by the sdk package itself

Reply:
we should be good
https://github.com/OffchainLabs/arb-token-bridge/search?q=rpcURL

Reply:
cc @spsjvc